sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -440,20 +443,15 @@ public void setReplayId(final @Nullable String replayId) {
     set(DSCKeys.REPLAY_ID, replayId);
   }
 
-  @ApiStatus.Internal
-  public void set(final @NotNull String key, final @Nullable String value) {
-    set(key, value, false);
```

Comment:
This was always called with `force=false`, so I just merged `set(key, value, force)` and `set(key, value)`.